app/javascript/components/search/results/ResultsPanel.js

```diff
@@ -15,38 +14,50 @@ const ResultsPanel = props => {
   const results = searchContext.results
   let panelContent
   if (searchContext.isError) {
-    panelContent =
-      <div className="error-panel">
-        <FontAwesomeIcon icon={faExclamationCircle}/>
-        Sorry, an error has occurred.  Support has been notified.
-        Please try again.
+    panelContent = (
+      <div className="error-panel  col-md-6 col-md-offset-3">
+        <FontAwesomeIcon
+          icon={faExclamationCircle}
+          style={{ 'margin-right': '5px' }}
```

Comment:
For now, we're trying to avoid inline CSS where possible.  Let's move this to a css file.  give this a className like "left-margin-icon" or something, because I suspect this will be a generally useful class.  Also, make the margin in ems rather than px so that it will scale with font resizes